* Do not set CC
* Adhere to CFLAGS and LDFLAGS, do not set optimisation flags


--- a/src/makefile	2004-12-12 00:51:29.000000000 +0100
+++ b/src/makefile	2010-05-07 03:26:38.000000000 +0200
@@ -24,8 +24,7 @@
 
 ### Compiler options #########################################################
 
-CC		= gcc
-CFLAGS          = -O2 -Wall -DAVRLANG=LANGUAGE_$(LANGUAGE)
+CFLAGS          += -Wall -DAVRLANG=LANGUAGE_$(LANGUAGE)
 CPPFLAGS	=
 CCFLAGS         = $(CFLAGS) $(CPPFLAGS)
 LINK            = -lm -lstdc++
@@ -39,7 +38,7 @@
 ### Executable ##############################################################
 
 $(EXENAME):	$(OBJS)
-		$(CC) $(CCFLAGS) $(OBJS) $(LINK) -o $@
+		$(CC) $(CCFLAGS) $(LDFLAGS) $(OBJS) $(LINK) -o $@
 
 ### Parser file #############################################################
 
